src/Tooltip/style.css

```diff
@@ -0,0 +1,97 @@
+.ola_tooltip {
+    display: inline-block;
+    position: relative;
+}
+.ola_tooltip-trigger {
+    list-style: none;
+    outline: 0;
+    &::-webkit-details-marker {
+        display: none;
+    }
+}
+.ola_tooltip-content {
+    --size: var(--size-5);
+
+    position: absolute;
+    filter: drop-shadow(var(--shadow-2)) drop-shadow(var(--shadow-line));
+    background: var(--white);
+    border-radius: var(--radius);
+    padding: var(--size-6);
+    width: var(--size-11);
+    box-sizing: border-box;
+    z-index: 5;
```

Comment:
The open/close animation looks buggy in Safari because `filter`. It can be fixed using `will-change: opacity, transform;` to improve the performance.

Reply:
Yeah! I didn't know that property.
Now it works.